ls/definition_test.go

```diff
@@ -24,7 +24,7 @@ type testingTB interface {
 }
 
 // newTestServer creates a new language server for testing
-func newTestServer(t testingTB, workspace string) *Server {
+func newTestServer(t testingTB, workspaces []string) *Server {
```

Comment:
More beautiful and less changes in the test files...
```suggestion
func newTestServer(t testingTB, workspaces ...string) *Server {
```